burner-ui/src/Pages/ActivityPage/ActivityPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HistoryEvent from '@burner-wallet/core/HistoryEvent';
+import { Box } from 'rimble-ui';
+import { BurnerContext, withBurner } from '../../BurnerProvider';
+import Page from '../../components/Page';
+import History from '../../data-providers/History';
+import HistoryListEvent from '../../components/HistoryListEvent';
+
+const ActivityPage: React.FC<BurnerContext> = ({ defaultAccount, actions }) => {
+  return (
+    <Page title="Activity" back>
+      <Box mt={24} padding="0 var(--page-margin)" as="section">
+        <History
+          account={defaultAccount}
+          render={(events: HistoryEvent[]) => events.slice(0, 3).map((event: HistoryEvent) => (
```

Comment:
Looks good. The first event shows a top border which doesn't look exactly right, but I can fix that by removing it via `:first-child {}` css pseudo-class, which I'll do when we merge into `develop`